pvacseq/lib/parse_output.py

```diff
@@ -188,7 +194,6 @@ def match_wildtype_and_mutant_entry_for_inframe_indel(result, mt_position, wt_re
             #We then check if the alternate best match epitope has more matching amino acids than the baseline best match epitope
             #If it does, we pick it as the best match
             if consecutive_matches_from_right > best_match_count:
```

Comment:
We still need to look at two possible matches for indels: the one that lines up at the before and the one that lines up after the mutation. We then need to determine which one is the better match. We could use the total match count for this instead of just the consecutive matches. 

It would be a bigger code  change to look at the total number of amino acids for this determination. Right now, once we determine that the eptiope to the right of the mutation is the better match we can from then on match all eptiopes further down the full sequence on the same side of the mutation, instead of continuing to check both options on the off chance that the one on the left of the mutation might have more total matching AAs.

But I'm happy to change it if we think it's more correct (which I kinda thing it is).